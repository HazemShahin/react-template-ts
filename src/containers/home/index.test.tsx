import * as React from 'react';
import { shallow } from 'enzyme';

import { Home } from '.';

jest.mock('react-inlinesvg');

describe('Home component', () => {
	it('should render successfully', () => {
		const tree = shallow(<Home />);

		expect(tree).toMatchSnapshot();
	});
});
