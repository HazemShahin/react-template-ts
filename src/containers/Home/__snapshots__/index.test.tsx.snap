// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render successfully 1`] = `
Array [
  <span
    className="isvg loading c-svg-icon"
  />,
  <div
    className="o-wrapper"
  >
    <button
      className="c-btn "
      onClick={[Function]}
    >
      -
    </button>
    <button
      className="c-btn "
      onClick={[Function]}
    >
      Async -
    </button>
    <small>
      0
    </small>
    <button
      className="c-btn "
      onClick={[Function]}
    >
      Async +
    </button>
    <button
      className="c-btn "
      onClick={[Function]}
    >
      +
    </button>
  </div>,
  <img
    alt="Redux Saga Logo"
    style={
      Object {
        "display": "block",
        "margin": "auto",
      }
    }
  />,
]
`;
